### PR TITLE
add experimental FROST/EdDSA signature combination

### PR DESCRIPTION
**Pull request by cairomassimo**
# Description

Adds support to combining FROST-type signature shares to the `crypto` package.

Relies on https://github.com/LIT-Protocol/lit-ecdsa-wasm-combine/pull/5.

NOTE: the way this functionality is exposed in the api is not ideal, and only temporary. (Namely, it was added to the `combineEcdsaShares` method, despite we are not dealing with ECDSA, and the input/output types do not conform to the signature of that method.)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/crypto/src/lib/crypto.ts [hunk 1]
```diff
@@ -270,14 +270,20 @@ export const combineEcdsaShares = (
         log('response from combine_signature', res);
         sig = JSON.parse(res);
         break;
+      case SIGTYPE.FrostEd25519Sha512v1:
+        res = ecdsaSdk.combine_signature(validShares, 4);
```
Comment (joshLong145) [inline packages/crypto/src/lib/crypto.ts:274]: Is there an update to the `combiner` that follows this new `FrostEd25519Sha512v1` if so can you link that as well in this pr?

Comment (cairomassimo) [reply packages/crypto/src/lib/crypto.ts:274]: Right! Here: https://github.com/LIT-Protocol/lit-ecdsa-wasm-combine/pull/5 (will add to the description as well)

**Comment by joshLong145** [comment]
@cairomassimo 
Can we start a feature branch for this work on https://linear.app/litprotocol/issue/LIT-2262/eddsa-and-schnorr
since this is being tracked as a project we can make a feature branch and merge it into that.

**Comment by cairomassimo** [comment]
Closing in favor of #341

**Pull request closed without merging**